Sources/Lotsawa/Chart.swift

```diff
@@ -47,17 +47,21 @@ extension Chart {
     /// It is arranged to avoid 64-bit alignment, since this will be combined into an `Entry`.
     /// You'd want to swap the elements for optimal performance on big-endian architectures.
     var storage: (
-      /// 16-bit dot position and the low 16 bits of the origin
-      dotPosition_originLow: UInt32,
-      /// The high 16 bits of the origin, 1 bit for isEarley, 14 bit transition or LHS symbol,
-      /// 1 bit for isCompletion.
-      originHi_isEarley_symbol_isCompletion: UInt32)
+      /// The low 16 bits of the origin and a 16-bit dot position.
+      // bits        | xxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxx |
+      // meaning     | originLow        | dotPosition      |
```

Comment:
Something like this?  It's markdown, so...
```suggestion
      /// 
      ///     bit         | 31 ... 16 | 15  ...   0 |
      ///     meaning     | originLow | dotPosition |
```

Reply:
Counting x's is no fun, and I can't tell which one is the MSB anyway.